<ui version="4.0" >
 <class>CustomActionBase</class>
 <widget class="QWidget" name="CustomActionBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Actions builder</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="icons.qrc" >:/icons/resources/bookmark.png</iconset>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>15</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonNew" >
            <property name="text" >
             <string>&amp;New</string>
            </property>
            <property name="icon" >
             <iconset resource="icons.qrc" >:/icons/resources/bookmark_add.png</iconset>
            </property>
            <property name="shortcut" >
             <string>Alt+N</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRename" >
            <property name="text" >
             <string>&amp;Rename</string>
            </property>
            <property name="icon" >
             <iconset resource="icons.qrc" >:/icons/resources/pencil.png</iconset>
            </property>
            <property name="shortcut" >
             <string>Alt+R</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonRemove" >
            <property name="text" >
             <string>R&amp;emove</string>
            </property>
            <property name="icon" >
             <iconset resource="icons.qrc" >:/icons/resources/remove.png</iconset>
            </property>
            <property name="shortcut" >
             <string>Alt+E</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonMoveUp" >
            <property name="layoutDirection" >
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text" >
             <string>Move &amp;up</string>
            </property>
            <property name="icon" >
             <iconset resource="icons.qrc" >:/icons/resources/1uparrow.png</iconset>
            </property>
            <property name="shortcut" >
             <string>Alt+U</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonMoveDown" >
            <property name="text" >
             <string>Move &amp;down</string>
            </property>
            <property name="icon" >
             <iconset resource="icons.qrc" >:/icons/resources/1downarrow.png</iconset>
            </property>
            <property name="shortcut" >
             <string>Alt+D</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QFrame" name="frame_3" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="labelActionName" >
             <property name="text" >
              <string>Name</string>
             </property>
             <property name="buddy" >
              <cstring>listWidgetNames</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="listWidgetNames" />
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="frame_2" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="textLabel1" >
             <property name="text" >
              <string>Insert commands to run:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEditAction" >
             <property name="toolTip" >
              <string>Insert commands to run. See handbook (F1) for some examples.</string>
             </property>
             <property name="acceptRichText" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QCheckBox" name="checkBoxRefreshAfterAction" >
               <property name="toolTip" >
                <string>If checked main view will be automatically refreshed when done</string>
               </property>
               <property name="text" >
                <string>Refresh &amp;view at the end</string>
               </property>
               <property name="shortcut" >
                <string>Alt+V</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxAskArgs" >
               <property name="toolTip" >
                <string>Check if you want to be prompted for command line arguments. Args will be appended after the first command only.</string>
               </property>
               <property name="text" >
                <string>A&amp;sk for command line arguments</string>
               </property>
               <property name="shortcut" >
                <string>Alt+S</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>101</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonOk" >
         <property name="text" >
          <string>&amp;Ok</string>
         </property>
         <property name="shortcut" >
          <string>Alt+O</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="icons.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButtonNew</sender>
   <signal>clicked()</signal>
   <receiver>CustomActionBase</receiver>
   <slot>pushButtonNew_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRename</sender>
   <signal>clicked()</signal>
   <receiver>CustomActionBase</receiver>
   <slot>pushButtonRename_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRemove</sender>
   <signal>clicked()</signal>
   <receiver>CustomActionBase</receiver>
   <slot>pushButtonRemove_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxRefreshAfterAction</sender>
   <signal>toggled(bool)</signal>
   <receiver>CustomActionBase</receiver>
   <slot>checkBoxRefreshAfterAction_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMoveUp</sender>
   <signal>clicked()</signal>
   <receiver>CustomActionBase</receiver>
   <slot>pushButtonMoveUp_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMoveDown</sender>
   <signal>clicked()</signal>
   <receiver>CustomActionBase</receiver>
   <slot>pushButtonMoveDown_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAskArgs</sender>
   <signal>toggled(bool)</signal>
   <receiver>CustomActionBase</receiver>
   <slot>checkBoxAskArgs_toggled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>CustomActionBase</receiver>
   <slot>pushButtonOk_clicked()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEditAction</sender>
   <signal>textChanged()</signal>
   <receiver>CustomActionBase</receiver>
   <slot>textEditAction_textChanged()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>395</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel" >
     <x>330</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetNames</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>CustomActionBase</receiver>
   <slot>listWidgetNames_currentItemChanged(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>129</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel" >
     <x>337</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
